### DISCUSSION TITLE
schedule is set but PR is created unexpected timing

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [renovatebot/renovate#8548] [Request Help] [answered]
[renovate.json](https://github.com/suzuki-shunsuke/test-renovate-2/blob/954557ae6eee3e7419a016babc0df4966e072ecb/renovate.json)

The schedule is

```json
["at 2:00 am on Mon, Thurs"]
```

And I pushed [a commit](https://github.com/suzuki-shunsuke/test-renovate-2/commit/6e4ec8e39bb01e181c2fa9b01d19a26c38b6bd1d) at 22:41 on Thursday (UTC), then a pull request was created at 22:48 on Thursday (UTC).

https://github.com/suzuki-shunsuke/test-renovate-2/pull/49

> Schedule: "at 2:00 am on Mon, Thurs" (UTC).

![image](https://user-images.githubusercontent.com/13323303/106968737-c863d400-678c-11eb-830c-233fbd38f8f0.png)

I expect the pull request isn't created until the next Monday.

Is this behavior expected?
Is the configuration wrong?

**Comment by viceice** [comment, marked as answer]
Two things:

* the `,` makes them 2 different schedules, so you have `at 2:00 am on Mon`  and `on Thurs`
* `at 2:00 am` is a bad idea, as you can't guarantee that renovate run exactly on that time. You should use at least a 2h time frame.

**Reply by suzuki-shunsuke** [reply]
Oh, I see. Thanks.

So if we want to schedule between 2 am and 4 am on Monday and Thursday, we should configure as the following, right?

```json
["after 2am and before 4am on Mon and Thurs"]
```

I'll test this setting.

**Reply by viceice** [reply]
not 100% sure, but yes i think so 😉

**Reply by suzuki-shunsuke** [reply]
I tested the above setting and it works well.
Thanks again.
